### DISCUSSION TITLE
Displaying image thumbnails in Collection layout

### DISCUSSION THREAD

**Discussion by justrealmilk** [directus/directus#7382] [Archive]
![image](https://user-images.githubusercontent.com/156681/129251570-fa66ad97-b574-4317-98c3-f6471aee9ee3.png)

What are the requirements for displaying image thumbnails in Collections layouts?

After comparing collections where thumbnails are and are not available (especially `directus_fields`), I can't determine what the trick is...

**Comment by rijkvanzanten** [comment]
Does your `cover` field have a display configured? The `image` display will preview a little thumbnail
